tests/manage/pv_services/test_no_space_left.py

```diff
@@ -112,14 +112,12 @@ def test_pvc_no_space_left(self, base_setup):
             'fs', size=self.pvc_size_gb, io_direction='write', runtime=60
         )
         log.info("Waiting for IO results")
-        catch_error = False
         try:
             self.pod_obj1.get_fio_results()
         except ex.CommandFailed as cf:
             if "No space left on device" not in str(cf):
```

Comment:
This will still fail on Cephfs right?

Reply:
fio is passing with cephfs, so CommandFailed error won't be raised.